### DISCUSSION TITLE
Does FanControl install graphics drivers?

### DISCUSSION THREAD

**Discussion by Tanz0rz** [Rem0o/FanControl.Releases#2457] [Q&A]
I have never had an AMD GPU in this computer, but AMD drivers are constantly being installed automatically by something. I tried to manually delete the DLLs from my system folder, but Windows is saying FanControl is using them. Does FanControl install these if it thinks it is necessary?

**Comment by Rem0o** [comment]
under the hood, it uses LHM which use https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/blob/331d334aef6939482ae0b5491865311b87de17ba/LibreHardwareMonitorLib/Interop/AtiAdlxx.cs#L42C12-L42C13

I don't know the exact dll gymnastic is does under the hood.
